mmv1/third_party/terraform/services/bigtable/resource_bigtable_authorized_view.go

```diff
@@ -0,0 +1,462 @@
+			Optional:    true,
+			Elem:        &schema.Schema{Type: schema.TypeString},
+			Description: `Base64-encoded individual exact column qualifiers of the column family to be included in the authorized view.`,
+		},
+		"qualifier_prefixes": {
+			Type:        schema.TypeSet,
+			Optional:    true,
+			Elem:        &schema.Schema{Type: schema.TypeString},
+			Description: `Base64-encoded prefixes for qualifiers of the column family to be included in the authorized view. Every qualifier starting with one of these prefixes is included in the authorized view. To provide access to all qualifiers, include the empty string as a prefix ("").`,
+		},
+	},
+}
+
+func ResourceBigtableAuthorizedView() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceBigtableAuthorizedViewCreate,
+		Read:   resourceBigtableAuthorizedViewRead,
+		Update: resourceBigtableAuthorizedViewUpdate,
+		Delete: resourceBigtableAuthorizedViewDestroy,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceBigtableAuthorizedViewImport,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(20 * time.Minute),
+			Update: schema.DefaultTimeout(20 * time.Minute),
+		},
+
+		CustomizeDiff: customdiff.All(
+			tpgresource.DefaultProviderProject,
+		),
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `The name of the authorized view. Must be 1-50 characters and must only contain hyphens, underscores, periods, letters and numbers.`,
+			},
+
+			"project": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: `The ID of the project in which the resource belongs. If it is not provided, the provider project is used.`,
+			},
+
+			"instance_name": {
+				Type:             schema.TypeString,
+				Required:         true,
+				ForceNew:         true,
+				DiffSuppressFunc: tpgresource.CompareResourceNames,
+				Description:      `The name of the Bigtable instance in which the authorized view belongs.`,
+			},
+
+			"table_name": {
+				Type:             schema.TypeString,
+				Required:         true,
+				ForceNew:         true,
+				DiffSuppressFunc: tpgresource.CompareResourceNames,
+				Description:      `The name of the Bigtable table in which the authorized view belongs.`,
+			},
+
+			"deletion_protection": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: validation.StringInSlice([]string{"PROTECTED", "UNPROTECTED"}, false),
+				Description:  `A field to make the authorized view protected against data loss i.e. when set to PROTECTED, deleting the authorized view, the table containing the authorized view, and the instance containing the authorized view would be prohibited. If not provided, currently deletion protection will be set to UNPROTECTED as it is the API default value.`,
```

Comment:
It may be important to document that this is deletion protection in the API/backend. `deletion_protection` uses the same name as other resource's fields that are Terraform-side deletion protection like https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#deletion_protection

Reply:
Done.